lib/minfeatures.nix

```diff
@@ -0,0 +1,16 @@
+let
+  features = [
+    {
+      description = "the `nixVersion` builtin";
+      condition = builtins ? nixVersion;
+    }
+    {
+      description = "`builtins.nixVersion` reports at least 2.18";
+      condition = builtins ? nixVersion && builtins.compareVersions "2.18" builtins.nixVersion != 1;
+    }
+  ];
+in
+# Exposes the missing features for evaluating Nixpkgs
+builtins.map ({ description, ... }: description) (
+  builtins.filter ({ condition, ... }: !condition) features
+)
```

Comment:
I wonder if it makes more sense to expose all features? That might make debugging/developing easier.

To avoid needing to filter by condition externally, we could return an attrset of `{ all, present, missing }`. Essentially `partition`ing the features list.